change_log/v9_0_0/FE-1761-pages-switch-fix.yaml

```diff
@@ -0,0 +1 @@
+Component Fix: Page slides would glitch due to dependency of the 'name' property (that stores page/slide number) of Button which was being removed from DOM. Depending how quickly button element would be removed from DOM Carousel slider would work or not.
```

Comment:
That description suggests that there was an issue with a component code and the issue was with a story.

Reply:
Agree with @grabkowski, would expect to see a description of the change made rather than what was wrong. The code changes look fine to me and I'll be happy to approve once you've updated